### PR TITLE
[Tadhg] Updated database names and changed Eureka Clients to Feign Cl…

### PR DESCRIPTION
**Pull request by tadhgryan2012**
…ients

## Summary
Changed database names so each service can access the database again
Updated Clients to be Feign clients to improve performance.

## How to verify
Test endpoints and database connections of all services.

## Related

### CODE REVIEW

**Comment by tadhgryan2012** [comment]
Tested the end points in messaging and it works, db in there works too.

**Review by AlB1111122 — DISMISSED** [review]
every service that makes a call to another needs to have a config class that looks like this: 
ackage CS4337.Project;

import org.springframework.cloud.client.loadbalancer.LoadBalanced;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@Configuration
public class Config {

  @Bean
  @LoadBalanced // This enables Eureka client for service discovery
  public RestTemplate restTemplate() {
    return new RestTemplate();
  }
}
In order to have eurikaservice actually resolve the urls. Then the instanciation of resttemplate needs to be autowired

  @Autowired private final RestTemplate restTemplate = new RestTemplate();

also i cant figure out how to fix this but for me instead of using my ip or localhost its using the docker container id as the server for the routing which does not allow requests to be sent

File: AuthService/src/main/resources/application.properties [hunk 1]
```diff
@@ -8,10 +8,13 @@ spring.security.oauth2.client.registration.google.redirect-uri=${GOOGLE_REDIRECT
 
 secret-key=${SECRET_KEY}
 
-spring.datasource.url=jdbc:mysql://mysql:3306/mydb
+spring.datasource.url=jdbc:mysql://mysql:3306/service
```
Comment (oOoOoOo0o) [inline AuthService/src/main/resources/application.properties:11]: Shouldn't just be service here

**Review by oOoOoOo0o — CHANGES_REQUESTED** [review]

File: AuthService/src/main/resources/application.properties (hunk 1 above)
Comment (tadhgryan2012) [reply AuthService/src/main/resources/application.properties]: Updated endpoint of the database to match docker-compose.yml

**Review by tadhgryan2012 — COMMENTED** [review]
Updated this to work now

**Review by oOoOoOo0o — APPROVED** [review]

**Pull request merged**